### PR TITLE
Add serverless documentation 

### PR DESCRIPTION
**Pull request by avelanarius**
Add a documentation describing how to connect to a ScyllaDB Cloud serverless database. The documentation is based on the existing "Quick start" manual:

https://java-driver.docs.scylladb.com/scylla-3.11.2.x/manual/index.html

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -187,7 +187,10 @@ def setup(app):
 
     # Replace DataStax links
     current_slug = os.getenv("SPHINX_MULTIVERSION_NAME", "stable")
-    replacements = {r'docs.datastax.com/en/drivers/java\/(.*?)\/': "java-driver.docs.scylladb.com/" + current_slug + "/api/"}
+    replacements = {
+        r'docs.datastax.com/en/drivers/java\/(.*?)\/': "java-driver.docs.scylladb.com/" + current_slug + "/api/",
+        r'java-driver.docs.scylladb.com\/(.*?)\/': "java-driver.docs.scylladb.com/" + current_slug + "/api/"
+    }
```
Comment (tzach) [inline docs/source/conf.py:193]: Is this change related to serverless ?

File: manual/serverless/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    ResultSet rs = session.execute("select release_version from system.local");  // (4)
+    Row row = rs.one();
+    System.out.println(row.getString("release_version"));                        // (5)
+} finally {
+    if (cluster != null) cluster.close();                                        // (6)
+}
+```
+
+1. the [Cluster] object is the main entry point of the driver. It holds the known state of the actual ScyllaDB cluster
```
Comment (tzach) [inline manual/serverless/README.md]:
```suggestion
1. The [Cluster] object is the main entry point of the driver. It holds the known state of the actual ScyllaDB cluster
```

File: manual/serverless/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+   (notably the [Metadata](metadata/)). This class is thread-safe, you should create a single instance (per target
+   ScyllaDB cluster), and share it throughout your application;
+2. [withScyllaCloudConnectionConfig] is a method that configures the cluster endpoints and credentials
+   to your ScyllaDB Cloud serverless cluster based on the YAML connection bundle you downloaded from ScyllaDB Cloud;
+3. the [Session] is what you use to execute queries. Likewise, it is thread-safe and should be reused;
```
Comment (tzach) [inline manual/serverless/README.md]:
```suggestion
3. The [Session] is what you use to execute queries. Likewise, it is thread-safe and should be reused;
```

File: manual/serverless/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+4. we use `execute` to send a query to Cassandra. This returns a [ResultSet], which is essentially a collection of [Row]
```
Comment (tzach) [inline manual/serverless/README.md]:
```suggestion
4. We use `execute` to send a query to Cassandra. This returns a [ResultSet], which is essentially a collection of [Row]
```

File: manual/serverless/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+   objects. On the next line, we extract the first row (which is the only one in this case);
+5. we extract the value of the first (and only) column from the row;
```
Comment (tzach) [inline manual/serverless/README.md]:
```suggestion
5. We extract the value of the first (and only) column from the row;
```

File: manual/serverless/README.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+6. finally, we close the cluster after we're done with it. This will also close any session that was created from this
```
Comment (tzach) [inline manual/serverless/README.md]:
```suggestion
6. Finally, we close the cluster after we're done with it. This will also close any session that was created from this
```

**Review by tzach — COMMENTED** [review]
Few minor comments
I only review the text, didnt test the code

**Comment by annastuchlik** [comment]
The content looks good, but the preview gives me some warnings.
@avelanarius Have you successfully run the preview? When I do it, I get:
image
With the following warnings:
\java-driver\docs\_source\contents.rst:1: WARNING: toctree glob pattern 'manual/*' didn't match any documents
\java-driver\docs\_source\contents.rst:1: WARNING: toctree glob pattern 'upgrade_guide/*' didn't match any documents
\java-driver\docs\_source\contents.rst:1: WARNING: toctree glob pattern 'faq/*' didn't match any documents
\java-driver\docs\_source\contents.rst:1: WARNING: toctree glob pattern 'changelog/*' didn't match any documents
